src/app/shared/components/featured/featured.component.html

```diff
@@ -0,0 +1,16 @@
+<ecl-featured
+  [isExtended]="isExtended"
+  [position]="alignment"
+>
+  <ecl-featured-item>
+    <ecl-media-container>
+      <img eclMediaContainerItem
+        src="{{ imageUrl }}"
+        alt="Lorem ipsum dolor sit amet">
```

Comment:
Also the alt text should be coming from an input

Reply:
this naming is a bit confusing, it makes you think that featured is a group containing items, i see that the eUI main tag is ecl-featured, but i would name our component app-ecl-featured-item

Reply:
the alt is coming from input now